product/views/ems_storage.yaml

```diff
@@ -9,10 +9,10 @@
 #
 
 # Report title
-title: Block Storage Managers
+title: Storage Managers
 
 # Menu name
-name: EmsBlockStorage
+name: BlockStorage
```

Comment:
This seems wrong - Not sure what this is supposed to be though.

Reply:
`EmsStorage` maybe?

Reply:
i don't really remember why i put `BlockStorage` , let me check :)